jetstream/engine/warmup_utils.py

```diff
@@ -44,34 +43,30 @@ def layout_params_and_compile_executables(
   any_prefill_engine = None
   any_prefill_params = None
 
-  prefill_executables = []
-  inserts_generate_executables = []
+  prefills_compiled = []
+  inserts_generate_compiled = []
 
   for i, pe in enumerate(prefill_engines):
     any_prefill_engine = pe
     any_prefill_params = prefill_params[i]
-    prefill_executable = initialize_prefill_jit_cache(
+    prefill_compiled = initialize_prefill_jit_cache(
         prefill_engine=pe,
         prefill_params=prefill_params[i],
         prefill_idx=i,
     )
-    prefill_executables.append(prefill_executable)
+    prefills_compiled.append(prefill_compiled)
 
   for i, ge in enumerate(generate_engines):
-    insert_executable, generate_executable = (
-        initialize_insert_generate_jit_cache(
-            prefill_engine=any_prefill_engine,
-            generate_engine=ge,
-            prefill_params=any_prefill_params,
-            generate_params=generate_params[i],
-            generate_idx=i,
-        )
-    )
-    inserts_generate_executables.append(
-        [insert_executable, generate_executable]
+    insert_generate_compiled = initialize_insert_generate_jit_cache(
+        prefill_engine=any_prefill_engine,
+        generate_engine=ge,
+        prefill_params=any_prefill_params,
+        generate_params=generate_params[i],
+        generate_idx=i,
     )
+    inserts_generate_compiled.append([insert_generate_compiled])
 
-  if prefill_executables and inserts_generate_executables:
+  if any(prefills_compiled) and any(inserts_generate_compiled):
```

Comment:
Sorry, I guess I mistake my previous comment, the condition should be all truth for both prefills_compiled and inserts_generate_compiled IIUC? Then we should use all() instead of any()

Reply:
Thanks, changed!